src/client/containers/BoardPageComponents/BoardAddCandidate/AddCandidate.js

```diff
@@ -16,6 +20,8 @@ export default async function AddCandidate(newCandidate, userId, boardId) {
     await postData(`/api/users/${userId}/boards/${boardId}/candidates`, {
       name: newCandidate.name,
     });
+    setBoardLoading(true);
```

Comment:
I assume you just want to set it to true here

Reply:
Yes, I just set the loading to false again to be able to see the component again in the browser to test it. Just removed it